core/api_handler.py

```diff
@@ -144,11 +145,15 @@ def upload_sequencing_run(sequencing_run):
         # set seq run to upload
         api_instance.set_seq_run_uploading(run_id)
 
+        current_project_number = 0
```

Comment:
What are these `current_project_number` and `current_sample_number` vars actually being used for?

Reply:
Removed the lines. They were missed when I was cleaning up the progress signals.